test/distributed/org/apache/cassandra/distributed/test/accord/NewSchemaTest.java

```diff
@@ -75,7 +75,7 @@ private static Query read(String ks, String table, List<ByteBuffer> keys)
         assert !keys.isEmpty();
         Txn.Builder builder = new Txn.Builder();
         for (int i = 0; i < keys.size(); i++)
-            builder.addLet("row" + i, new Select.Builder().withWildcard().withTable(ks, table).withColumnEquals("pk", keys.get(i)));
+            builder.addLet("row" + i, new Select.Builder().wildcard().table(ks, table).value("pk", keys.get(i)));
```

Comment:
in this case `wildcard` isn't needed, but it does protect the builder from something adding selections later on... because we not holding the reference after this line we really could remove, but left it for now.